test/fixtures/permission/fs-read.js

```diff
@@ -278,4 +346,11 @@ const regularFile = __filename;
     permission: 'FileSystemRead',
     resource: path.toNamespacedPath(blockedFile),
   }));
-}
+  assert.throws(() => {
+    fs.statfs(blockedFileURL, () => {});
+  }, common.expectsError({
+    code: 'ERR_ACCESS_DENIED',
+    permission: 'FileSystemRead',
+    resource: path.toNamespacedPath(blockedFile),
+  }));
+}
```

Comment:
Missing LF at end of file. I am surprised that the linter doesn't complain.

Reply:
Oh, perhaps because it's in `test/fixtures`.

Reply:
Yeah, lint doesn't pass on fixtures. Wanna me to put LF anyway?

Reply:
No strong opinion either way :)